js/index.js

```diff
@@ -57,27 +57,41 @@ document.addEventListener("DOMContentLoaded", () => {
         ************ PROJECTS  *************
         ====================================
      */
-  const githubRequest = new XMLHttpRequest();
-
-  githubRequest.addEventListener("load", function () {
-    if (githubRequest.readyState === 4 && githubRequest.status === 200) {
-      let repositories = JSON.parse(this.response);
-      const projectSection = document.getElementById("projects");
-      const projectList = projectSection.querySelector("ul");
-
-      for (let i = 0; i < repositories.length; i++) {
-        const project = document.createElement("li");
-        const repoLink = document.createElement("a");
-        repoLink.href = repositories[i].html_url;
-        repoLink.textContent = repositories[i].name;
-
-        project.appendChild(repoLink);
-        projectList.appendChild(project);
-      }
+  const projectSection = document.getElementById("projects");
+  const projectList = projectSection.querySelector("ul");
+  const url = "https://api.github.com/users/wessstt/repos";
+
+  fetch(url)
+    .then(checkStatus)
```

Comment:
This is very readable! Note that this depends on [function hoisting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function#hoisting)... Since the definitions are coming out of order, it could be a little harder to trace, but it's really fine either way.